qbatch

```diff
@@ -170,4 +202,8 @@ if __name__ == "__main__":
         os.chmod(script, os.stat(script).st_mode | stat.S_IXUSR)
         if dry_run:
             continue
-        subprocess.call(['qsub', script])
+        if system == 'sge' or system == 'pbs':
+            subprocess.call(['qsub', script])
+        else:
+            print "Launching Local Jobscript, saving output to logdir/{}.log".format(job_name)
```

Comment:
That's actually how it is here, only the "local mode" creates a logfile, as there is no scheduler to do the logging for us.

--logdir or --logfile is a good idea. I forgot I had TODO'd that in my head.